fabric-transfer-api-v1/src/testmod/java/net/fabricmc/fabric/test/transfer/ingame/TransferTestInitializer.java

```diff
@@ -63,6 +74,74 @@ public void onInitialize() {
 		ItemStorage.SIDED.registerForBlocks((world, pos, state, be, direction) -> TrashingStorage.ITEM, Blocks.OBSIDIAN);
 		// And diamond ore blocks are an infinite source of diamonds! Yay!
 		ItemStorage.SIDED.registerForBlocks((world, pos, state, be, direction) -> CreativeStorage.DIAMONDS, Blocks.DIAMOND_ORE);
+
+		CommandRegistrationCallback.EVENT.register((dispatcher, registryAccess, environment) -> {
+			dispatcher.register(
+					CommandManager.literal("fabric_insertintoheldstack")
+							.then(CommandManager.argument("stack", ItemStackArgumentType.itemStack(registryAccess))
+									.then(CommandManager.argument("count", LongArgumentType.longArg(1))
+											.executes(context -> {
+												ItemVariant variant = ItemVariant.of(ItemStackArgumentType.getItemStackArgument(context, "stack")
```

Comment:
Is there any check against trying to insert blank? Or maybe allowing that is a feature?

Reply:
Isn't inserting/extracting blank explicitly prohibited by transfer API?
If you're talking about the commands, the reason they don't have checks is because they're only meant for testing this feature, so trying to insert blank and crashing would be the user's fault